src/libraries/System.Runtime.Numerics/src/System/Numerics/BigInteger.cs

```diff
@@ -857,15 +859,15 @@ public static BigInteger GreatestCommonDivisor(BigInteger left, BigInteger right
                 Debug.Assert(right._bits != null);
                 return left._sign != 0
                     ? BigIntegerCalculator.Gcd(right._bits, NumericsHelpers.Abs(left._sign))
-                    : new BigInteger(right._bits, null, negative: false);
+                    : new BigInteger(right._bits, false);
```

Comment:
When passing literal Boolean arguments, prefer to use `argName: true` or `argName: false` to help code clarity.

Reply:
Fixed across entire code base.